lib/src/base_dialog.dart

```diff
@@ -29,6 +29,7 @@ class BaseGiffyDialog extends StatefulWidget {
     @required this.cornerRadius,
     @required this.buttonRadius,
     @required this.entryAnimation,
+    this.onCancelButtonPressed,
```

Comment:
Just make it @required and we are good to go

Reply:
Thanks dude, I just did, rgds